### PR TITLE
Improve ntfy notifications

### PR DESCRIPTION
**Pull request by mtelgkamp**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [x] I have read and understand the pull request rules.

# Description

Improve ntfy notifications.
- use tags `red_circle` for down and `green_circle` for up
- increase priority for down alert by 1 if not already max
- add monitor name and status to title
- use heartbeat msg as Message
- add monitor url as action

## Type of change

- Other (improve notification provider)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [x] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

screenshot ntfy alert

### CODE REVIEW

File: server/notification-providers/ntfy.js [hunk 1]
```diff
@@ -14,11 +15,45 @@ class Ntfy extends NotificationProvider {
...
+                "priority": priority,
+                "title": monitorJSON.name + " " + status + " [Uptime-Kuma]",
+                "tags": tags,
+                "actions": [
+                    {
+                        "action": "view",
+                        "label": "Open " + monitorJSON.name,
+                        "url": monitorJSON.url
+                    }
+                ]
```
Comment (Computroniks) [inline server/notification-providers/ntfy.js:56]: Can we keep the trailing commas here

**Review by Computroniks — CHANGES_REQUESTED** [review]

File: server/notification-providers/ntfy.js (hunk 1 above)
Comment (mtelgkamp) [reply server/notification-providers/ntfy.js]: I added the trailing comma here and in line 25-

**Comment by mabed-fr** [comment]
Hello

Good job !

Why not replace "keyword" with the word actually searched by the probe/check?

**Comment by mtelgkamp** [comment]
> Good job !

Thank you

> Why not replace "keyword" with the word actually searched by the probe/check?

I use the variable `heartbeatJSON.msg` for the message, so changes to the text are not part of the notification, but need to be done where the variable is filled, to change the text for all notification providers.

**Pull request merged**